### ISSUE TITLE
Part Design → Tree View

### ISSUE THREAD

**Issue by kaktusus** [FreeCAD/FreeCAD-translations#170]
```
OS: Debian GNU/Linux bookworm/sid (KDE/plasma)
Word size of FreeCAD: 64-bit
Version: 0.20.29177 (Git) AppImage
Build type: Release
Branch: (HEAD detached at 0.20)
Hash: 68e337670e227889217652ddac593c93b5e8dc94
Python 3.10.5, Qt 5.12.9, Coin 4.0.0, Vtk 9.1.0, OCC 7.5.3
Locale: Polish/Poland (pl_PL)
```

Pop-up message has no translation.

![image](https://user-images.githubusercontent.com/39754351/174428690-4810be62-af19-45f0-a4d4-bd234576df02.png)

1.  `Sketch with malformed constraints
Please remove the following malformed constraint:` - no string in Crowdin

    https://crowdin.com/translate/freecad/all/en-pl?filter=basic&value=0#q=Sketch%20with%20malformed%20constraints

**Comment by Kuzma30** [comment]
Strings defined here,
https://github.com/FreeCAD/FreeCAD/blob/fb1a7af3605ed86631afabb1eb77ae305e418d98/src/Mod/Sketcher/App/SketchObject.cpp#L201-L223
https://github.com/FreeCAD/FreeCAD/blob/fb1a7af3605ed86631afabb1eb77ae305e418d98/src/Mod/Sketcher/App/SketchObject.cpp#L7476-L7519

@chennes @yorikvanhavre Can we translate such strings?

**Comment by chennes** [comment]
Yes, absolutely, those can be translated, they just need to be put in calls to `tr()`.

**Comment by Kuzma30** [comment]
@chennes I use QT_TR_NOOP macro
Please review this
https://github.com/Kuzma30/FreeCAD/commit/ab58d7c58ba63b0c2323af26d94d94c995806d80

**Comment by chennes** [comment]
I don't think that will work, but I haven't looked carefully at the details of `DocumentObjectExecReturn`, maybe it calls a translate function in its argument. Did you test?

**Comment by Kuzma30** [comment]
@chennes Just test with translation. You are right. This don't work...

**Comment by chennes** [comment]
I'll take care of doing it this morning, give me a couple hours.

**Comment by kaktusus** [comment]
still needs attention.

```
OS: Debian GNU/Linux trixie/sid (KDE/plasma)
Word size of FreeCAD: 64-bit
Version: 0.21.0.33458 (Git) AppImage
Build type: Release
Branch: master
Hash: 8b89dd29d2382c72d738c52e8974216130ffa9db
Python 3.10.12, Qt 5.15.8, Coin 4.0.0, Vtk 9.1.0, OCC 7.6.3
Locale: Polish/Poland (pl_PL)
Installed mods: 
  * Curves 0.5.0
  * sheetmetal 0.2.56
  * Reinforcement
  * BIM 2021.12.0
  * Rocket 3.1.0
  * kicadStepUpMod 10.19.4
  * fasteners 0.4.56
```
